addons/Rakugo/lib/systems/Parser.gd

```diff
@@ -65,7 +65,8 @@ var parser_regex :={
 	# jump label
 	JUMP = "^jump (?<label>{NAME})( if (?<expression>.+))?$",
 	# for setting Rakugo variables
-	SET_VARIABLE = "^(?<lvar_name>{VARIABLE}) = ((?<text>{STRING})|(?<number>{NUMERIC})|(?<rvar_name>{VARIABLE}))$",
+	ASSIGNMENT = "^(=|\\+=|\\-=|\\*=|\\/=|\\%=)",
```

Comment:
Please move it in var Regex. var parser_regex is only for RkScript instructions.

Reply:
Okey I will move it, I didn't know you had such assumptions in the code.

Reply:
Is done along a go.